src/components/navigation/menu/PhoneMenu/ToggleButton.js

```diff
@@ -32,9 +32,9 @@ const StyledToggleButton = styled(ToggleButton)`
   right: 30px;
   top: 18px;
   z-index: 9991;
-
+  ${FONT_FAMILY}
   label {
-    font-size: 12px;
+    font-size: 15px;
```

Comment:
@shakersinthedark font-size from the design system?

Reply:
Nope, it's specific to this component as it's an edge case